README.md

```diff
@@ -12,7 +12,7 @@ A Client library for OAuth2 which uses JWT for the generation of access tokens.
               .subject("myapp ")
               .build();
 
-  TokenSource tokenSource = config.tokenSource();
+  TokenSource tokenSource = TokenSources.reusableTokenSource(new OkHttpTokenSource(config));
```

Comment:
As bearer authorizartion header is used by the interceptor is clear that it's a bearer.

I'll change it to BearerJwtAuthenticationInterceptor is it make it concrete that JWT is used.